packages/paste-website/src/pages/components/switch/index.mdx

```diff
@@ -74,6 +75,18 @@ A Switch is an interactive binary control. It should be used in forms when an "o
 text to describe the purpose of the Switch. The label text shouldn't change when the state of the Switch changes. Help text
 can also be used to provide additional guidance.
 
+When using Switch, be sure to provide immediate results that indicate confirmation of the interaction. Users shouldn't need to perform an additional action (such as a "Submit" button) to confirm or save their Switch input. For example, when a user turns on a "Receive notifications" Switch, they should start receiving notifications immediately upon performing the action.
```

Comment:
suggestion: Add a link to Checkbox and say that people can use Checkbox if the change is not immediate: https://paste.twilio.design/components/checkbox#about-checkbox <- Match language used here.